docs/guide.md

```diff
@@ -767,6 +767,29 @@ efficient platform for running Python projects.
 In case you do not want to include DevOps in your project, you can do so
 by selecting the option `None` (this is the default option).
 
+
+## Automation Tools
+
+An automation tool is software or a platform designed to automate repetitive 
+tasks, processes, or workflows that are traditionally performed manually.
+These tools are used to streamline and optimize various operations, reduce 
+human intervention, increase efficiency, and minimize errors.
+Currently OSL is using `make` and `Makim`
```

Comment:
Improve it, Here it should be SciCookie.

Reply:
@ayeankit, could you open a follow-up PR to change OSL to SciCokkie, as pointed by @anavelyz, please?

thanks @anavelyz for catching that!

Reply:
sure !!